File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const size = nums.length;
        for(let i=0;i<size-1;i++){
            for(let j=i+1;j<size;j++){
                if(nums[i]==nums[j])
                    return true;
            }
        }
        return false;
    }

}

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-1.py
import heapq

class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        self.heap = []

        for n in nums:
            heapq.heappush(self.heap, n)

            if len(self.heap) > k:
                heapq.heappop(self.heap)

        return self.heap[0]